cmd/lifecycle/creator.go

```diff
@@ -300,3 +301,16 @@ func (c *createCmd) WriteableRegistryImages() []string {
 	}
 	return writeableImages
 }
+
+func (c *createCmd) populateRunImageIfNeeded() error {
```

Comment:
This is almost the exact function that we have in the `analyzer`. Can we move it to a common place?

Reply:
We can't have a common function because `populateRunImageIfNeeded()` updates the receiving struct. 

Eventually, as we move logic out of the `cmd` package, we might have the `<some phase>Cmd` structs implement an interface, and this could be one method on it.

Reply:
```suggestion
func (c *createCmd) populateRunImage() error {
```

nit: I think dropping the `IfNeeded` reads better to me.

Reply:
Please see f2ec72b